cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -3033,6 +3033,26 @@ pAlwaysAbstain =
     , Opt.help "Abstain from voting on all proposals."
     ]
 
+pVoteAnchor :: Parser (AnchorUrl, VoteTextSource)
+pVoteAnchor = (,)
+  <$> (AnchorUrl <$> pUrl "vote-anchor-url" "vote anchor URL")
+  <*> pVoteTextSource
+
+pVoteTextSource :: Parser VoteTextSource
+pVoteTextSource =
+  asum
+    [ VoteTextSourceText
```

Comment:
Shouldn't it be either `vote-anchor-url` and `vote-anchor-text` _or_ `vote-anchor-file`?

Reply:
`vote-anchor-file` sounds like its the entire anchor i.e url and datahash.

Reply:
Ah the parser confused me. You should rename it to `vote-anchor-text-file` otherwise it implies it the entire anchor.